### PR TITLE
Enrollment CoC extension

### PR DESCRIPTION
**Pull request by eanders**
This will set EnrollmentCoc to match ProjectCoC CoC Code if the project only has one CoC, otherwise, it will set it to blank.

### CODE REVIEW

File: drivers/hmis_csv_importer/app/models/hmis_csv_importer/hmis_csv_cleanup/force_project_enrollment_coc.rb [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+      enrollment_coc_scope.find_each do |e_coc|
+        # Don't replace blanks
+        next if e_coc.CoCCode.blank?
+
+        # If we have a single CoC from the project, this will set it,
+        # if not, this will set it to nil
+        coc_code = coc_for(e_coc.ProjectID)
+        next if coc_code.blank?
+
```
Comment (awisema) [inline drivers/hmis_csv_importer/app/models/hmis_csv_importer/hmis_csv_cleanup/force_project_enrollment_coc.rb:20]: Maybe add to only touch ones that are wrong?
```ruby
next if coc_code == e_coc.CoCCode
```

File: drivers/hmis_csv_importer/spec/models/importer/twenty_twenty/force_valid_coc_spec.rb [hunk 2]
```diff
@@ -55,7 +55,7 @@ def setup(with_cleanup:)
     end
 
     import_hmis_csv_fixture(
-      'drivers/hmis_csv_importer/spec/fixtures/files/twenty_twenty/cleanup_move_ins',
+      'drivers/hmis_csv_importer/spec/fixtures/files/twenty_twentytwo/cleanup_move_ins',
```
Comment (awisema) [inline drivers/hmis_csv_importer/spec/models/importer/twenty_twenty/force_valid_coc_spec.rb:58]: Should we have separate fixtures for each extension?

**Review by awisema — APPROVED** [review]

File: drivers/hmis_csv_importer/app/models/hmis_csv_importer/hmis_csv_cleanup/force_project_enrollment_coc.rb (hunk 1 above)
Comment (eanders) [reply drivers/hmis_csv_importer/app/models/hmis_csv_importer/hmis_csv_cleanup/force_project_enrollment_coc.rb:20]: Good idea

File: drivers/hmis_csv_importer/spec/models/importer/twenty_twenty/force_valid_coc_spec.rb (hunk 2 above)
Comment (eanders) [reply drivers/hmis_csv_importer/spec/models/importer/twenty_twenty/force_valid_coc_spec.rb:58]: Hmm, I bungled this a bit.  Pushed a fix with new fixture and tests to cover the case where we can't determine the CoC.

**Pull request merged**